packages/app/src/app/components/Preview/DevTools/Tests/TestElement/elements.ts

```diff
@@ -22,8 +22,7 @@ export const Actions = styled.div`
 `;
 
 export const TestName = styled.div`
-  padding: 0.25rem;
-  padding-left: 20px;
+  padding: 0.25rem 0.25rem 0.25rem 20px;
```

Comment:
In general I'd like the explicit separate rule for `padding-left`. I don't mind this change, but I don't think future usages of `padding-left` etc need to be refactored.